### DISCUSSION TITLE
Is Laravel "session()" global helper function can be used in the custom WebHooks handler?

### DISCUSSION THREAD

**Discussion by jialeee17** [def-studio/telegraph#531] [Q&A] [answered]
Hello guys, I'm creating a Telegram bot using this package. I have a custom WebHooks handler and I want to use session for storing some data but it didn't work out. The session was not created. Here's my code...

`
    public function connect()
    {
        try {
            session(['data' => 123]);
        } catch (Exception $e) {
            Log::channel('telegram')->error($e->getMessage());
            $this->chat->html(ChatService::error())->send();
        }
    }
`

The "session()" is available where I can use it to check for data existence, but I couldn't use it to save data into session. Is anyone knows why? Thanks in advance.

**Comment by fabio-ivona** [comment, marked as answer]
Telegram server doesn't sends session info, so this solution isn't feasible

I think you can use telegraph storage system, please refer to to the docs for more info
